onboarding/source/controller.c

```diff
@@ -26,7 +27,9 @@
 
 /* USER CODE BEGIN */
 // define config for the light timer
-
+#define LIGHT_CONTROLLER_NAME         "light timer" 
```

Comment:
These macro names are misleading. The "light controller" is a task, whereas this is a timer to send an event to that task.